apps/gallery/locales/gallery.en-US.properties

```diff
@@ -100,11 +100,14 @@ back-button.ariaLabel = Back
 
 # Edit crop buttons
 crop-options.ariaLabel = Crop
-crop-revert.ariaLabel = Revert
 crop-aspect-free.ariaLabel = Custom aspect ratio
 crop-aspect-portrait.ariaLabel = Portrait aspect ratio
 crop-aspect-landscape.ariaLabel = Landscape aspect ratio
 crop-aspect-square.ariaLabel = Square aspect ratio
+crop-aspect-more.ariaLabel = More aspect ratios
+crop-aspect-3x4.ariaLabel = 3:4
+crop-aspect-4x3.ariaLabel = 4:3
+crop-aspect-16x9.ariaLabel = 16:9
```

Comment:
For these ariaLabel strings, I suspect that it would be better to be explicit and say "3:4 aspect ratio", etc.

Do these strings also set the text for the menu items, or do you need to define 

crop-aspect-3x4=3:4

and the other strings separately? 

If you're going to make the ariaLabel strings more verbose than the visible strings, then you'll need separate properties, of course.

I suppose you might also localize these (both the strings and the aria labels) to say "3:4 portrait" and "4:3 landscape" and "16:9 landscape" since there isn't an icon that shows the orientation of the rectangle.

And finally, I'd guess that these new strings will mean that uplifting this to 2.2 is out of the question. I assume that the l10n deadline has passed, but I don't actually know that for sure.